### ISSUE TITLE
Metadata pollution

### ISSUE THREAD

**Issue by grawlinson** [cyrilis/epub-gen#7]
I'm currently using this to rapidly generate ePub files and I've noticed that the metadata somehow becomes polluted.

If I open a generated ePub in Calibre (or any ePub reader/editor), and look at the metadata, the IDs field shows something like this:
```
urn:urn:uuid:<PROJECT DIRECTORY>/node_modules/epub-gen/tempDir/246c4f08-e94f-4377-a197-a3fcca958615
```

Having the folder structure of the project exposed isn't desirable, it would be nice if it showed only the UUID.

**Comment by cyrilis** [comment]
Hi @grawlinson ,

Thanks. This seems a bug, I've fixed it in https://github.com/cyrilis/epub-gen/commit/a921b1d2dd3e2921e40f13040b4a0ffc4fd1d7e3. 

Sorry for the pollution.

**Comment by grawlinson** [comment]
That's OK, happy to help.

**Comment by grawlinson** [comment]
I've just submitted a pull request that hopefully fixes all pollution issues.

**Comment by cyrilis** [comment]
Hi @grawlinson ,

Thanks, I should removed them before open-sourced.  Merged #10.
